server/repository/src/db_diesel/cold_storage_type_row.rs

```diff
@@ -0,0 +1,94 @@
+use super::{
+    cold_storage_type_row::cold_storage_type::dsl as cold_storage_type_dsl, StorageConnection,
+};
+use crate::{repository_error::RepositoryError, Upsert};
+use crate::{ChangeLogInsertRow, ChangelogRepository, ChangelogTableName, RowActionType};
+
+use diesel::prelude::*;
+
+table! {
+    cold_storage_type (id) {
+        id -> Text,
+        name -> Text,
+        min_temperature -> Double,
+        max_temperature -> Double,
+    }
+}
+
+#[derive(
+    Clone, Queryable, Insertable, AsChangeset, Debug, PartialEq, Default, serde::Serialize,
+)]
+#[diesel(table_name = cold_storage_type)]
+pub struct ColdStorageTypeRow {
+    pub id: String,
+    pub name: String,
+    pub min_temperature: f64,
+    pub max_temperature: f64,
+}
+
+pub struct ColdStorageTypeRowRepository<'a> {
+    connection: &'a StorageConnection,
+}
+
+impl<'a> ColdStorageTypeRowRepository<'a> {
+    pub fn new(connection: &'a StorageConnection) -> Self {
+        ColdStorageTypeRowRepository { connection }
+    }
+
+    pub fn upsert_one(&self, row: &ColdStorageTypeRow) -> Result<i64, RepositoryError> {
+        diesel::insert_into(cold_storage_type_dsl::cold_storage_type)
+            .values(row)
+            .on_conflict(cold_storage_type_dsl::id)
+            .do_update()
+            .set(row)
+            .execute(self.connection.lock().connection())?;
+        self.insert_changelog(row, RowActionType::Upsert)
```

Comment:
Actually - I don't think this needs to be in the changelog - its OG central only data (will only be pulled from OG, we won't create any changes within OMS that would need to sync)

Reply:
You mean remove the whole `insert_changelog` function?